system.c

```diff
@@ -295,6 +295,49 @@ uint8_t system_execute_line(char *line)
           if (line[++char_counter] != 0)
             return(STATUS_INVALID_STATEMENT);
           break;
+        case 'B': // Turn braking on or off
+	{
```

Comment:
I will fix the indentation though :). My editor decided a tab was appropriate here...